### ISSUE TITLE
Aortic cusps calcification

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6574]
**Preferred term label:**  Aortic cusps calcification

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Aortic valve calcification HP:0004380

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
How is this different from aortic valve calcification? @eswietlik

**Comment by eswietlik** [comment]
@pnrobinson Cusps and annulus constitute a valve. One can have calcified cusps without calcification of the annulus and vice versa. The distinction is important in many clinical situations such as TAVI. Severe annulus calcification with small annulus may be a contraindication for the procedure whereas calcification of the cusps is an indication for it. Also,  calcification of annulus (but not mitral or aortic cusps) correlates with atherosclerosis in other vascular beds. Similarly, LVOT and annulus calcification are independent predictors of aortic regurgitation after device implantation.

**Comment by pnrobinson** [comment]
@eswietlik The literature I am looking seems to assume that if you say "aortic valve calcification" and if you have annular calfication you say "aortic annulus calcification". Thus, ``Aortic cusp calcification`` seems to be more like a synoynm than a child of ``Aortic valve calcification``.

**Comment by pnrobinson** [comment]
adding term. It seemed that this is more commonly used "Aortic valve leaflet calcification", so I made it the primary label the the other the synonym.
